packages/terra-search-field/src/SearchField.jsx

```diff
@@ -273,6 +273,7 @@ class SearchField extends React.Component {
       <div {...customProps} className={searchFieldClassNames}>
         <div className={cx('input-group')}>
           <input
+            {...additionalInputAttributes}
```

Comment:
can we add a test case for this?

Reply:
Added jest to show that custom class does not override terra styles. c92ed69